tka/sig_test.go

```diff
@@ -175,6 +175,82 @@ func TestSigNested_DeepNesting(t *testing.T) {
 	}
 }
 
+func TestSigCredential(t *testing.T) {
```

Comment:
Is there a test that the credential can't be used more than once?

Reply:
Theres nothing wrong with using a credential multiple times for authorizing different node keys. Unless you mean a check that SigCredential cant nest another signature?

Reply:
> SigCredential describes a signature over a single-use public key

I feel like that implies it's only usable once.

Anyway that makes preauth keys really powerful and I guess generation and distribution of those will need to be done in a way that control never sees them.

Reply:
100%, we definitely need to make sure control never sees them. I was thinking that, after someone generates a preauth key in the admin panel, they can do a `tailscale lock key-wrap` to generate and encode the `SigCredential` and its private key around the preauth key.

I'll update the language to remove the confusion that the public key can only be used once, rather than that the generated public key is intended to only be used for a single SigCredential.